docs/source/workflows/base/relax.rst

```diff
@@ -139,6 +141,8 @@ Only ``structure`` and ``engines`` can be specified as a positional argument, al
 
         input_generator.get_spin_types()
 
+.. _relax-ref-wc:
```

Comment:
This is correct, but if you compile the docs and open the link, you see that in the current format, the navigation bar of the docs covers at least three lines below the pointer. This is the reason of my choice.
Is it maybe a thing one can fix with some Sphinx options?

Reply:
I see what you mean, but I don't think this should be the solution. I don't think now it is much clearer why we end up in the middle of some other input. @chrisjsewell maybe you know how to fix this. What is happening with this anchor is that when clicked, it does correcly position the page at this entry, but the header covers the first few lines. We use the exact same theme as `aiida-core` so the same problem must occur there. I imagine this is probably a css problem since the header is floating over the main content and so the page is correctly positioned with respect to the position of the anchor it is just that it doesn't realize there is a header there. How can we take the height of the header into account and have it push the page further down by that height?